javalanglib/src/main/scala/java/lang/Integer.scala

```diff
@@ -53,24 +53,32 @@ object Integer {
   final val MIN_VALUE = -2147483648
   final val MAX_VALUE = 2147483647
   final val SIZE = 32
+  final val BYTES = 4
 
   @inline def valueOf(intValue: scala.Int): Integer = new Integer(intValue)
   @inline def valueOf(s: String): Integer = valueOf(parseInt(s))
-
-  @inline def valueOf(s: String, radix: Int): Integer =
-    valueOf(parseInt(s, radix))
+  @inline def valueOf(s: String, radix: Int): Integer = valueOf(parseInt(s, radix))
 
   @inline def parseInt(s: String): scala.Int = parseInt(s, 10)
 
-  def parseInt(s: String, radix: scala.Int): scala.Int = {
+  def parseInt(s: String, radix: scala.Int): scala.Int =
+    parseIntImpl(s, radix, signed = true)
+
+  @inline def parseUnsignedInt(s: String): scala.Int = parseUnsignedInt(s, 10)
+
+  def parseUnsignedInt(s: String, radix: scala.Int): scala.Int =
+    parseIntImpl(s, radix, signed = false)
+
+  @inline
+  private def parseIntImpl(s: String, radix: scala.Int, signed: scala.Boolean): scala.Int = {
```

Comment:
Line too long. 

Also, the callers of this method should be marked `@noinline`.